Makefile

```diff
@@ -125,76 +125,76 @@ dev: dev-install dev-stop
 	  /opt/mesosphere/bin/dcos-net-env console
 
 ##
-## DC/OS E2E
+## miniDC/OS
 ##
 
-DCOS_DOCKER_TRANSPORT ?= docker-exec
-DCOS_DOCKER_CUSTOM_VOLUME ?= "$(BASE_DIR):$(BASE_DIR):rw"
-DCOS_DOCKER_CLUSTER_ID ?= default
-DCOS_DOCKER_MASTERS ?= 1
-DCOS_DOCKER_AGENTS ?= 1
-DCOS_DOCKER_PUBLIC_AGENTS ?= 0
-DCOS_DOCKER_NODE ?= master_0
-DCOS_DOCKER_WEB_PORT ?= 443
-
-dcos-docker-create:
-	@ dcos-docker inspect \
-	      --cluster-id $(DCOS_DOCKER_CLUSTER_ID) \
+MINIDCOS_TRANSPORT ?= docker-exec
+MINIDCOS_CUSTOM_VOLUME ?= "$(BASE_DIR):$(BASE_DIR):rw"
+MINIDCOS_CLUSTER_ID ?= default
+MINIDCOS_MASTERS ?= 1
+MINIDCOS_AGENTS ?= 1
+MINIDCOS_PUBLIC_AGENTS ?= 0
+MINIDCOS_NODE ?= master_0
+MINIDCOS_WEB_PORT ?= 443
+
+minidcos-create:
+	@ minidcos docker inspect \
+	      --cluster-id $(MINIDCOS_CLUSTER_ID) \
 	      > /dev/null 2> /dev/null || \
-	( dcos-docker create dcos_generate_config.sh \
-	      --transport $(DCOS_DOCKER_TRANSPORT) \
-	      --masters $(DCOS_DOCKER_MASTERS) \
-	      --agents $(DCOS_DOCKER_AGENTS) \
-	      --public-agents $(DCOS_DOCKER_PUBLIC_AGENTS) \
-	      --cluster-id $(DCOS_DOCKER_CLUSTER_ID) \
-	      --custom-volume $(DCOS_DOCKER_CUSTOM_VOLUME) \
-	      $(DCOS_DOCKER_OPTS) && \
-	  dcos-docker wait \
-	      --transport $(DCOS_DOCKER_TRANSPORT) \
-	      --cluster-id $(DCOS_DOCKER_CLUSTER_ID) \
+	( minidcos docker create dcos_generate_config.sh \
+	      --transport $(MINIDCOS_TRANSPORT) \
+	      --masters $(MINIDCOS_MASTERS) \
+	      --agents $(MINIDCOS_AGENTS) \
+	      --public-agents $(MINIDCOS_PUBLIC_AGENTS) \
+	      --cluster-id $(MINIDCOS_CLUSTER_ID) \
+	      --custom-volume $(MINIDCOS_CUSTOM_VOLUME) \
+	      $(MINIDCOS_OPTS) && \
+	  minidcos docker wait \
+	      --transport $(MINIDCOS_TRANSPORT) \
+	      --cluster-id $(MINIDCOS_CLUSTER_ID) \
 	      --skip-http-checks )
 
-dcos-docker-destroy:
+minidcos-destroy:
 	@ docker ps \
-	    --filter label=dcos-e2e-web-id=$(DCOS_DOCKER_CLUSTER_ID) \
+	    --filter label=minidcos-web-id=$(MINIDCOS_CLUSTER_ID) \
 	    --format '{{.ID}}' \
-	| xargs docker kill > /dev/null
-	@ dcos-docker destroy --cluster-id $(DCOS_DOCKER_CLUSTER_ID)
+	| while read id; do [ -z "$id" ] || docker kill $id; done > /dev/null
+	@ minidcos docker destroy --cluster-id $(MINIDCOS_CLUSTER_ID)
 
-dcos-docker-web: dcos-docker-create
+minidcos-web: minidcos-create
 	@ docker ps \
-	    --filter label=dcos-e2e-web-id=$(DCOS_DOCKER_CLUSTER_ID) \
-	    --filter label=dcos-e2e-web-port=$(DCOS_DOCKER_WEB_PORT) \
+	    --filter label=minidcos-web-id=$(MINIDCOS_CLUSTER_ID) \
```

Comment:
As of miniDC/OS release, the labels have changed, but they don't have `minidcos` prefixes. It looks like `dcos-e2e.cluster_id` and `dcos-e2e.web-port` are what we should use now.

```
$ docker ps --format '{{.ID}}' | xargs docker inspect | jq '.[] | .Config.Labels + {name: .Name}'        
{
  "dcos_e2e.cluster_id": "default",
  "dcos_e2e.node_type": "agent",
  "dcos_e2e.variant": "oss",
  "dcos_e2e.workspace_dir": "/var/folders/70/8cktdr690m7_n95n__x9r2p80000gp/T/a81214c5050845f5b7617a14a87a68dd",
  "org.label-schema.build-date": "20181205",
  "org.label-schema.license": "GPLv2",
  "org.label-schema.name": "CentOS Base Image",
  "org.label-schema.schema-version": "1.0",
  "org.label-schema.vendor": "CentOS",
  "name": "/dcos-e2e-default-9890a-agent-0"
}
{
  "dcos_e2e.cluster_id": "default",
  "dcos_e2e.node_type": "master",
  "dcos_e2e.variant": "oss",
  "dcos_e2e.workspace_dir": "/var/folders/70/8cktdr690m7_n95n__x9r2p80000gp/T/a81214c5050845f5b7617a14a87a68dd",
  "org.label-schema.build-date": "20181205",
  "org.label-schema.license": "GPLv2",
  "org.label-schema.name": "CentOS Base Image",
  "org.label-schema.schema-version": "1.0",
  "org.label-schema.vendor": "CentOS",
  "name": "/dcos-e2e-default-9890a-master-0"
}
```

Reply:
I do not understand why you picked `dcos-e2e.cluster_id` (with a hyphen on dcos-e2e) instead of `dcos_e2e.cluster_id`(with underscore) as we see on the json above.

Reply:
Oops, you are right. Here are the correct label names: `dcos_e2e.cluster_id` and `dcos_e2e.web_port`.